CMGTools/TTHAnalysis/python/analyzers/susyCore_modules_cff.py

```diff
@@ -414,26 +493,66 @@
     collectionPostFix = "",
     )
 
-metNoHFAna = cfg.Analyzer(
-    METAnalyzer, name="metNoHFAnalyzer",
-    metCollection     = "slimmedMETsNoHF",
-    noPUMetCollection = "slimmedMETsNoHF",
-    copyMETsByValue = False,
+metAnaScaleUp = cfg.Analyzer(
+    METAnalyzer, name="metAnalyzerScaleUp",
+    metCollection     = "slimmedMETs",
+    noPUMetCollection = "slimmedMETs",    
+    copyMETsByValue = True,
     doTkMet = False,
     doMetNoPU = True,
     doMetNoMu = False,
     doMetNoEle = False,
     doMetNoPhoton = False,
-    recalibrate = False,
+    recalibrate = False, # or "type1", or True
     applyJetSmearing = False, # does nothing unless the jet smearing is turned on in the jet analyzer
-    old74XMiniAODs = False,   # can't be true, since MET NoHF wasn't there in old 74X MiniAODs
-    jetAnalyzerPostFix = "",
+    old74XMiniAODs = False, # set to True to get the correct Raw MET when running on old 74X MiniAODs
+    jetAnalyzerPostFix = "_jecUp",
     candidates='packedPFCandidates',
     candidatesTypes='std::vector<pat::PackedCandidate>',
     dzMax = 0.1,
-    collectionPostFix = "NoHF",
+    collectionPostFix = "_jecUp",
     )
 
+metAnaScaleDown = cfg.Analyzer(
+    METAnalyzer, name="metAnalyzerScaleDown",
+    metCollection     = "slimmedMETs",
+    noPUMetCollection = "slimmedMETs",    
+    copyMETsByValue = True,
+    doTkMet = False,
+    doMetNoPU = True,
+    doMetNoMu = False,
+    doMetNoEle = False,
+    doMetNoPhoton = False,
+    recalibrate = False, # or "type1", or True
+    applyJetSmearing = False, # does nothing unless the jet smearing is turned on in the jet analyzer
+    old74XMiniAODs = False, # set to True to get the correct Raw MET when running on old 74X MiniAODs
+    jetAnalyzerPostFix = "_jecDown",
+    candidates='packedPFCandidates',
+    candidatesTypes='std::vector<pat::PackedCandidate>',
+    dzMax = 0.1,
+    collectionPostFix = "_jecDown",
+    )
+
+#metNoHFAna = cfg.Analyzer(
```

Comment:
Is metNoHF fully deprecated for all datasets and usecases? 
If not, should it still be there.
@mariadalfonso